app/views/school/devices/cannot_order.html.erb

```diff
@@ -2,17 +2,22 @@
 <%- content_for :title, title %>
 <%- content_for :before_content do %>
   <% breadcrumbs([
-    { t('page_titles.school_home') => school_home_path },
-    title,
+    { 'Home' => school_home_path },
+    'Order devices',
```

Comment:
Can't we just change line 1 to:

```
<%- title = t('page_titles.school_order_devices') %>
```

and then leave this line and the h1 as is?

Reply:
No. The page title showing in `<title>` needs to remain the same. The "Order devices" in caption and breadcrumb is for orientation.